src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/NetStandardShims.cs

```diff
@@ -22,6 +22,14 @@ internal static unsafe int GetByteCount(this Encoding encoding, ReadOnlySpan<cha
             }
         }
 
+        internal static unsafe int GetBytes(this Encoding encoding, string str, Span<byte> destination)
+        {
+            if (str is null)
+                throw new ArgumentNullException(nameof(str));
+
+            return GetBytes(encoding, str.AsSpan(), destination);
+        }
```

Comment:
I don't see an Encoding.GetBytes(string, span).  Is this just trying to overcome the lack of an implicit conversion on older TFMs?  Because if it's just about the lack of an implicit conversion, I don't think the null-check is correct.